client/client-kv.c

```diff
@@ -0,0 +1,215 @@
+#include <stdlib.h>
```

Comment:
Why is this file included here? You're submitting server code. This is made obsolete by #58